packages/database/prisma/migrations/20251028181449_add_totalVotes_poll/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "public"."Poll" ADD COLUMN     "totalVotes" INTEGER NOT NULL DEFAULT 0;
```

Comment:
Could you add SQL logic in order to update `totalVotes` from current pollVotes ?